include/misc/sem.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ *
+ * @brief public sys_sem APIs.
+ */
+
+#ifndef ZEPHYR_INCLUDE_MISC_SEM_H_
```

Comment:
@wentongwu include/misc/ is deprecated.
Need to move this to include/sys/ instead.

Reply:
@andrewboie done